src/main/java/com/google/sps/data/Election.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.HashSet;
+
+/** An item on a todo list. */
+public final class Election {
+
+  private final String name;
+  private final String scope;
+  private final HashSet<long> positions;
+  private final String date;
+  private final HashSet<long> propositions;
+
+  public Election(String name, String scope, String date) {
```

Comment:
I may have mentioned this before: I'm okay with having a simple data container class like this given the time constaints of the project. But for a professional project, we would usually enforce that you use the Builder pattern:
https://refactoring.guru/design-patterns/builder/java/example

Usually autogenerated to reduce boilerplate using AutoValue:
https://github.com/google/auto/blob/master/value/userguide/index.md

You can just ACK (acknowledge) this comment.

Reply:
ACK